app/vmagent/README.md

```diff
@@ -60,6 +60,17 @@ Example command line:
 /path/to/vmagent -promscrape.config=/path/to/prometheus.yml -remoteWrite.url=https://victoria-metrics-host:8428/api/v1/write
 ```
 
+Example of minimal scrape configuration for `-promscrape.config` argument:
```

Comment:
Have you considered pointing to https://github.com/VictoriaMetrics/VictoriaMetrics/blob/master/deployment/docker/prometheus.yml instead?

Reply:
sounds good let me change